cli/kata-check_ppc64le.go

```diff
@@ -55,6 +62,17 @@ func hostIsVMContainerCapable(details vmContainerCapableDetails) error {
 		return err
 	}
 
+	text, err := getFileContents(details.cpuInfoFile)
+	if err != nil {
+		return err
+	}
+
+	if strings.Contains(text, "POWER8") {
+		if !isSMTOff() {
+			return fmt.Errorf("SMT is not Off. %s", failMessage)
+		}
+	}
```

Comment:
It's better to move this chunk under 
```
 	if count == 0 {
 		return nil
 	}
```

So that the judgement of `count == 0` can be near the assignment.

Reply:
I think we missed one tiny niggle - + 1 here for @WeiZhang555 comment - we should move the module check call to down underneath the cpuinfo/SMT check code I think. Right now the module check is split up by the cpuinfo/SMT code.

(sorry for the niggle - thanks for sticking with this @nitkon !)

Reply:
@grahamwhaley : Fixed!